purview/api/v1.py

```diff
@@ -75,24 +76,28 @@ def get_resource_story(resource, uid):
 
     results_unordered = {}
     if forward_query:
+        print("Forward Query:", forward_query)
         results_unordered = query_neo4j(forward_query)
 
     if backward_query:
         results_unordered.update(query_neo4j(backward_query))
 
-    results = []
+    results = OrderedDict()
     curr_label = 'BugzillaBug'
     while curr_label:
         if curr_label in results_unordered:
-            # Have each index of results be a list with the 0 index being the resource name
-            # and the the rest of the indexes being resources
-            results_unordered[curr_label].insert(0, curr_label)
-            results.append(results_unordered[curr_label])
+            if 'data' not in results:
```

Comment:
The results dictionary could just be initialized with this and you'd avoid this if statement.